pkg/cmd/pr/list/http.go

```diff
@@ -24,7 +24,7 @@ const fragment = `fragment pr on PullRequest {
 }`
 
 func listPullRequests(httpClient *http.Client, repo ghrepo.Interface, filters prShared.FilterOptions, limit int) (*api.PullRequestAndTotalCount, error) {
-	if filters.Author != "" || filters.Assignee != "" || filters.Search != "" {
+	if filters.Author != "" || filters.Assignee != "" || filters.Search != "" || len(filters.Labels) > 0 {
```

Comment:
I may have missed this in a previous PR but why does specifying `Author` or `Assignee` require the search API for pull requests but not for issues?

Reply:
Because the `Repository.issues` GraphQL connection allows filtering by author and assignee, but `Repository.pullRequests` doesn't.